pages/contribute-to-the-protocol.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Contribute to the Protocol
+keywords: governance, security program, Bitmark Upgrade Proposal
+last_updated: 
+sidebar: mydoc_sidebar
+permalink: /contribute-to-the-protocol
+folder: 
+---
```

Comment:
It feels like this section has a lot about Bitmark governance, which isn't really on topic for "Contribute to the Protocol".

I'd suggest either breaking this section up (into governance [or about-the-company] and contributing). Or maybe renaming it, if there's something that's more obviously about both topics.